tests/unit/modules/boto_iot_test.py

```diff
@@ -98,6 +98,7 @@ def _has_required_boto():
                       ruleDisabled=True)
 
 
+@skipIf(True, 'These tests are pegging the CPU and eating all available RAM')
 class BotoIoTTestCaseBase(TestCase):
```

Comment:
@kraney would you mind revisiting these to see what's going wrong?
You can trigger this by running all unittests `python tests/runtests.py -u`.